src/readabilityScore/Index.java

```diff
@@ -0,0 +1,34 @@
+package readabilityScore;
+
+public enum Index {
+    // Corresponds to Automated readability index
+    ARI {
+        @Override
+        IndexCalculator getCalculator() {
+            return new AutomatedReadabilityIndexCalculator();
+        }
+    },
+    // Corresponds to Flesch–Kincaid readability tests
+    FK {
```

Comment:
We could add a name to each enum instance, e.g. `FK("Flesch–Kincaid readability tests")`, then we can remove the comments here and also we can use this name for displaying the results